src/scripts/mutual_checker.js

```diff
@@ -32,8 +32,7 @@ const addIcons = function (postElements) {
     const postAttribution = postElement.querySelector(postAttributionSelector);
     if (postAttribution === null) { return; }
 
-    const blogLink = postAttribution.querySelector('a');
-    const blogName = blogLink?.textContent;
+    const blogName = postAttribution.textContent;
```

Comment:
```suggestion
    const blogName = postAttribution.textContent.trim();
```

I've been burned by this too many times before (see: Blacklist in XKit 7).